packages/mask/src/plugins/Avatar/Application/NFTListPage.tsx

```diff
@@ -0,0 +1,210 @@
+import { makeStyles } from '@masknet/theme'
+import {
+    ChainId,
+    createERC721Token,
+    ERC721TokenDetailed,
+    EthereumTokenType,
+    SocketState,
+    useCollectibles,
+    useImageChecker,
+} from '@masknet/web3-shared-evm'
+import { Box, Button, Skeleton, Typography } from '@mui/material'
+import { uniqBy } from 'lodash-unified'
+import { useCallback, useState } from 'react'
+import { useI18N } from '../locales'
+import { AddNFT } from '../SNSAdaptor/AddNFT'
+import { NFTImage } from '../SNSAdaptor/NFTImage'
+import type { TokenInfo } from '../types'
+
+const useStyles = makeStyles()((theme) => ({
+    root: {},
+    skeleton: {
+        width: 97,
+        height: 97,
+        objectFit: 'cover',
+        borderRadius: '100%',
+        boxSizing: 'border-box',
+        padding: 6,
+        margin: theme.spacing(0.5, 1),
+    },
+    skeletonBox: {
+        marginLeft: 'auto',
+        marginRight: 'auto',
+    },
+    error: {
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+        margin: 'auto',
+    },
+    button: {
+        textAlign: 'center',
+        paddingTop: theme.spacing(1),
+        display: 'flex',
+        justifyContent: 'space-around',
+        flexDirection: 'row',
+    },
+    gallery: {
+        display: 'flex',
+        flexFlow: 'row wrap',
+        overflowY: 'auto',
+        '&::-webkit-scrollbar': {
+            display: 'none',
+        },
+        justifyContent: 'space-between',
+    },
+}))
+
+interface NFTListPageProps {
+    chainId: ChainId
+    address: string
+    tokenInfo?: TokenInfo
+    tokens: ERC721TokenDetailed[]
+    onSelect?: (token: ERC721TokenDetailed) => void
+}
+
+export function NFTListPage(props: NFTListPageProps) {
+    const { classes } = useStyles()
+    const { address, onSelect, chainId, tokenInfo } = props
+    const t = useI18N()
+    const { data: collectibles, error, retry, state } = useCollectibles(address, chainId)
+    const [selectedToken, setSelectedToken] = useState<ERC721TokenDetailed | undefined>(
+        tokenInfo
+            ? createERC721Token(
+                  {
+                      name: '',
+                      address: tokenInfo.address,
+                      chainId,
+                      symbol: '',
+                      type: EthereumTokenType.ERC721,
+                  },
+                  {},
+                  tokenInfo.tokenId,
+              )
+            : undefined,
+    )
+
+    const onChange = (token: ERC721TokenDetailed) => {
+        if (!token) return
+        setSelectedToken(token)
+        onSelect?.(token)
+    }
+
+    const LoadStatus = Array.from({ length: 8 })
+        .fill(0)
+        .map((_, i) => (
```

Comment:
Create a range of number:

```suggestion
    const LoadStatus = Array.from({ length: 8 }, (_, index) => index)
        .map((i) => (
```

Or just use range from lodash

```suggestion
    const LoadStatus = range(8)
        .map((i) => (
```